client/Lykke.Service.Assets.Client/ServiceCollectionExtensions.cs

```diff
@@ -9,7 +10,7 @@ public static class ServiceCollectionExtensions
         public static void RegisterAssetsClient(this IServiceCollection services, AssetServiceSettings settings)
         {
             services
-                .AddSingleton<IAssetsService>(x => new AssetsService(settings.BaseUri, settings.Handlers));
+                .AddSingleton<IAssetsService>(x => new AssetsService(settings.BaseUri, new HttpClient()));
```

Comment:
Should we use HttpClientFactory?
https://www.stevejgordon.co.uk/introduction-to-httpclientfactory-aspnetcore

Reply:
we can use it later, now we should fix it ASAP